tests/unit/package_managers/yarn/test_main.py

```diff
@@ -48,9 +48,12 @@ class YarnVersions(Enum):
     YARN_V4_RC1 = semver.VersionInfo(4, 0, 0, prerelease="rc1")
     YARN_V4 = semver.VersionInfo(4, 0, 0)
 
+    YARN_V5_RC1 = semver.VersionInfo(5, 0, 0, prerelease="rc1")
+    YARN_V5 = semver.VersionInfo(5, 0, 0)
```

Comment:
Are these constants used anywhere ?

Reply:
Actually, yes. If you look at the `unsupported` method of this class, you'll see that all class attributes are used to calculated the list of unsupported versions.